plugins/lenovo-thinklmi/meson.build

```diff
@@ -0,0 +1,28 @@
+if get_option('plugin_uefi_capsule')
+
+cargs = ['-DG_LOG_DOMAIN="FuPluginLenovoThinkLmi"']
```

Comment:
do we maybe want to mock this up for a self test? e.g. set up a `tests/thinklmi/attributes/BootOrderLock/current_value`

Reply:
done